config/kernel-monitor.json

```diff
@@ -0,0 +1,31 @@
+{
+	"logPath": "/log/kern.log",
+	"bufferSize": 50,
+	"rules": [
+		{
+			"type": "temporary",
+			"reason": "OOMKilling",
+			"pattern": "Kill process \\d+ (.+) score \\d+ or sacrifice child\\nKilled process \\d+ (.+) total-vm:\\d+kB, anon-rss:\\d+kB, file-rss:\\d+kB"
+		},
+		{
+			"type": "temporary",
+			"reason": "TaskHung",
+			"pattern": "task \\S+:\\w+ blocked for more than \\w+ seconds\\."
+		},
+		{
+			"type": "permanent",
```

Comment:
Should all those permanent issues are introduced as NodeConditionTypes? I am ok to do that with following PRs later.